iothub/device/src/Transport/AmqpIoT/AmqpIoTTransport.cs

```diff
@@ -171,6 +171,18 @@ private async Task<ClientWebSocket> CreateClientWebSocketAsync(Uri websocketUri,
                     websocket.Options.ClientCertificates.Add(_amqpTransportSettings.ClientCertificate);
                 }
 
+                // Support for RemoteCertificateValidationCallback for ClientWebSocket is introduced in .NET Standard 2.1
+#if NETSTANDARD2_1
+                if (_amqpTransportSettings.RemoteCertificateValidationCallback != null)
+                {
```

Comment:
And what if _amqpTransportSettings.RemoteCertificateValidationCallback == null; is it safe to ignore the value of websocket.Options.RemoteCertificateValidationCallback ?

Reply:
If the application doesn't supply a validation callback then we don't set it